parm/config/gfs/config.prepoceanobs

```diff
@@ -15,6 +15,10 @@ export OBS_YAML=${OBS_LIST}
 # ocean analysis needs own dmpdir until standard dmpdir has full ocean obs
 export DMPDIR=@DMPDIR@
 
+# For BUFR2IODA json and python scripts
+export JSON_TMPL_DIR=${HOMEgfs}/sorc/gdas.cd/parm/ioda/bufr2ioda
```

Comment:
```suggestion
export JSON_TMPL_DIR=${HOMEgfs}/parm/gdas/ioda/bufr2ioda
```

Reply:
@aerorahul Trying to address this before opening the new PR - is the idea to link the yaml files from `${HOMEgfs}/sorc/gdas.cd/parm/ioda/bufr2ioda` to `${HOMEgfs}/parm/gdas/ioda/bufr2ioda`?

Reply:
Yes.
All files from `sorc/gdas.cd` should be linked in the appropriate location in `HOMEgfs/`.  The scripts, yamls, etc. should reference them using `EXECgfs` or `PARMgfs` or `SCRgfs` or `USHgfs` instead of `HOMEgfs/sorc/gdas.cd/`+`parm` or `scripts` or `ush`.